CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-rc.0
+
+<!-- generated comparing v5.0.0-beta.5..next -->
+
+_Sep 1, 2021_
+
+A big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Renamed packages to `@mui/*`. This is part of our effort to expand the scope of the library beyond Material Design. Most of the effort on v5 was about customizability. More details about the [plan](https://github.com/mui-org/material-ui/issues/27825) and the [why](https://github.com/mui-org/material-ui/discussions/27803)
```

Comment:
```suggestion
- 🎉 Renamed packages to `@mui/*` as part of rebranding the company, following the strategy of expanding the library scope beyond Material Design. For more details about it, check the [GitHub discussion](https://github.com/mui-org/material-ui/discussions/27803). 
```

The excerpt _"Most of the effort on v5 was about customizability"_ doesn't seem to make sense to be in this sentence. Although scope renaming is happening in v5, it doesn't have anything objectively related to customizability but rather to a broader strategy of growing out of MD (customizability being a part of that). Makes sense?!